Setup/ComputerControl/ComputerControl-Mac.md

```diff
@@ -17,9 +14,30 @@ If you have any questions regarding the documentation or find any errors in the
 
 Warning: if you don't have software development experience and deviate from the guide steps, you are at your own risk.
 
+## Part 1: Install Pre-requisites
 ### Install XCode App
 
-Go to Mac App Store App, install XCode App. This is a program made by Apple for software development. It also comes with basic software tools to build **SerialPrograms**. So it's safe to install it first.
+This is a program made by Apple for software development. It also comes with basic software tools to build **SerialPrograms**. So it's safe to install it first.
+
+1. Go to Mac App Store App: Click the `` Apple Menu at the top left of the screen. Click `App Store` to open.
```

Comment:
The character before "Apple Menu" doesn't seem to show up properly for me. I assume it's the Apple logo?

Reply:
Yep, it's probably visible on Apple devices but I've removed it just in case.